README.md

```diff
@@ -9,14 +9,40 @@ https://github.com/chrisjshull/homebridge-nest#using-a-google-account
 
 These four items needs to be included into config.json. An example of config.json can be find as _config.json.
 
-## Install
+## Local Install
 After clone the repo, change directory in to the folder.
 ```
 openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout ./selfsigned.key -out selfsigned.crt
 npm install
 npm run start
 ```
-open a browser and go to the following url: https://[IP]:5000.
+open a browser and go to the following url: https://127.0.0.1:5000.
+
+Default credentials:
+```
+user: admin
+pass: admin1
+```
+
+## Docker Install
+Build image
+```
+docker build -t nest-camera-site:latest .
+```
+
+Star container:
+```
+docker run --name nest-camera-site -d -p 5000:5000 -v [/path/to/your/nest_folder]:/nest nest-camera-site:latest
+```
+
+open a browser and go to the following url: https://127.0.0.1:5000.
+
+Default credentials:
+
+user: admin
+pass: admin1
+
+If you want to change crendentials delete file `database.json` and rebuild image.
```

Comment:
I am not very familiar with docker. But is a default username and password necessary for it. If not, I think let the user create a strong password by default would be a safer choice.

Reply:
This is because it's requesting user input at the start.

If you can suggest a command line with no user input that can generate the file/credentials on the fly, we can remove the defaults.

Anyway, it's common practice to provide default credentials and a why to change them!

Reply:
Got it. I guess a registration page in the future is a better solution. Let's do the default for now.